backend/src/test/java/io/castles/core/controller/GameControllerTest.java

```diff
@@ -48,6 +48,7 @@
     void setup() {
         GameSettings gameSettings = GameSettings.from(GameLobbySettings.builder().gameMode(GameMode.DEBUG).build());
         game = new Game(UUID.randomUUID(), gameSettings, Set.of(new Player("P1")), server.eventHandler());
+        game.initialize();
```

Comment:
i hope this is not breaking anything. As i understand initialization is just moving the games state from `START` to `DRAW`. As i need the draw state for the `shouldReturnDrawnTile()`, i changed it here.

Reply:
In the other test I was following a more decoupled test logic that would assert on a mocked `GameService`. If you want to align this line should be absolete but you'd need to add a `Mockito.when(gameService.getDrawnTile(any(), any()).thenReturn(drawnTile)` in the test you added. (just wrote this line without checking it though, but either this or something similar is needed)

Reply:
This shouldn't be necessary if you don't call `game.drawTile` in the test, which should be the way to go.

Reply:
I pushed a commit